ios/PurchasesHybridCommon/PurchasesHybridCommon/PaywallProxy.swift

```diff
@@ -216,4 +214,51 @@ extension PaywallProxy: PaywallViewControllerDelegate {
 
 }
 
+// MARK: - Deprecations
+
+@available(iOS 15.0, *)
+extension PaywallProxy {
```

Comment:
We can probably remove all these. I don't think we gain anything keeping them. I know it's a public SDK, but I'd be surprised if anyone was actually using these.

Reply:
Hmm, yes... https://github.com/RevenueCat/purchases-hybrid-common/pull/666/files#diff-3a3cff3944b3db826f3f47f7e980ca36ad3f9589beac3ba0aabc52f5f5a28b29R105 is also breaking

Reply:
Hmm, I'm not opposed... But in that case, I would probably stop using semver in PHC and use some different versioning system. If we are using semver and making a breaking change, I think we should still make a major (which is probably ok since it doesn't force us to make a major in the hybrids)

Reply:
Yeah that's what I was thinking too.